client_options.go

```diff
@@ -47,3 +47,13 @@ func WithoutRetries() ClientOptionFunc {
 		return nil
 	}
 }
+
+// WithCustomLimiter injects a custom rate limiter to the client.
+func WithCustomLimiter(limiter RateLimiter) ClientOptionFunc {
```

Comment:
I try to keep these options ordered in alphabetical order, so if you could update that...

Reply:
o yeah I can do that.

Reply:
done.